src/stash/src/postgres.rs

```diff
@@ -628,12 +647,8 @@ impl Stash {
                 // can't accidentally have the same epoch, nonce pair (especially risky if the
                 // current epoch has been bumped exactly once, then gets recreated by another
                 // connection that also bumps it once).
-                let row = tx
-                    .query_one(
-                        "UPDATE fence SET epoch=epoch+1, nonce=$1 RETURNING epoch",
-                        &[&self.nonce.to_vec()],
-                    )
-                    .await?;
+                let epoch_lower_bound = epoch_lower_bound.unwrap_or(NonZeroI64::MIN).get();
+                let row = tx.query_one(&format!("UPDATE fence SET epoch=GREATEST(epoch+1, $1), nonce=$2 RETURNING epoch"), &[&epoch_lower_bound, &self.nonce.to_vec()]).await?;
```

Comment:
```suggestion
                let row = tx.query_one("UPDATE fence SET epoch=GREATEST(epoch+1, $1), nonce=$2 RETURNING epoch", &[&epoch_lower_bound, &self.nonce.to_vec()]).await?;
```

Reply:
Weird that the linter missed this, it usually catches these.